sql/core/v1.2/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilterSuite.scala

```diff
@@ -255,7 +255,8 @@ class OrcFilterSuite extends OrcTest with SharedSparkSession {
       checkFilterPredicate($"_1" >= timestamps(3), PredicateLeaf.Operator.LESS_THAN)
 
       checkFilterPredicate(Literal(timestamps(0)) === $"_1", PredicateLeaf.Operator.EQUALS)
-      checkFilterPredicate(Literal(timestamps(0)) <=> $"_1", PredicateLeaf.Operator.NULL_SAFE_EQUALS)
+      checkFilterPredicate(Literal(timestamps(0)) <=> $"_1", PredicateLeaf.Operator
+                           .NULL_SAFE_EQUALS)
```

Comment:
If you don't mind, shall we split like the following?
```
      checkFilterPredicate(Literal(timestamps(0)) <=> $"_1",
        PredicateLeaf.Operator.NULL_SAFE_EQUALS)
```

Reply:
sure thing!